app/models/concerns/bulkrax/export_behavior.rb

```diff
@@ -26,25 +23,6 @@ def hyrax_record
       @hyrax_record ||= ActiveFedora::Base.find(self.identifier)
     end
 
-    def write_files
```

Comment:
exporting the file sets is now done from within the `write_files` method in the csv_parser (same as the bagit parser). it was necessary because each zip will need it's own "files" folder, instead of adding them after the fact.